src/features/DebugSession.ts

```diff
@@ -114,6 +124,87 @@ export class DebugSessionFeature extends LanguageClientConsumer
         ];
     }
 
+    public async createTemporaryIntegratedConsole(session: DebugSession) {
```

Comment:
Can you type this function? It's `undefined | typeof(this.tempSessionDetails)` whatever that thing's type is.